libraries/godot-iap/Makefile

```
@@ -241,11 +244,38 @@ clean-all: clean
 	@rm -rf $(IOS_GDEXT_DIR)/Package.resolved
 	@echo "$(GREEN)✓ Deep clean complete$(NC)"
 
+# Install a clean source template before every export. This avoids stale
+# resources when the local Godot version changes and applies the minimum AGP
+# required by the AndroidX dependencies shipped through openiap-google.
+prepare-android-export-template:
```

checkmake (0.3.2)

[warning] Target body for "prepare-android-export-template" exceeds allowed length of 5 lines (22).

(maxbodylength)